google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -1776,10 +1776,11 @@ def default_acl
         #     p.add "roles/owner", "user:owner@example.com"
         #   end
         #
-        def policy force: nil
+        def policy force: nil, requested_policy_version: nil
```

Comment:
What happens if the data is currently at version 3 and we request version 1. Do we get "downgraded" data, or does it raise an error?

Reply:
@dazuma This behavior will be specified (currently commented out) [in the acceptance test](https://github.com/googleapis/google-cloud-ruby/blob/b5266fed39fefb2d68b0e0f7a8c20431d6ac2dd2/google-cloud-storage/acceptance/storage/bucket_policy_test.rb#L97) once our projects are whitelisted (@jkwlui is working on it). The error message from the service is: "Requested policy version (1) cannot be less than the existing policy version (3)."

```rb
# expect { bucket_policy_v3.policy requested_policy_version: 1 }.must_raise Google::Cloud::Error
```